src/domains/transaction/pages/SendTransfer/SendTransfer.tsx

```diff
@@ -327,32 +160,36 @@ export const SendTransfer = () => {
 		abortReference.current = new AbortController();
 
 		const { network, senderAddress } = getValues();
+		if (!network) {
+			return;
+		}
```

Comment:
```suggestion
		assertNetwork(network);
```
i think the network always has value here because it has been checked [here](https://github.com/PayvoHQ/wallet/blob/bd6ebdd7a1034d007e51ad414dfb9e36cf68262a/src/domains/transaction/pages/SendTransfer/SendTransfer.tsx#L192)